shotgun_api3/lib/httplib2/__init__.py

```diff
@@ -1,16 +1,39 @@
 from .. import six
 
-# import the proper implementation into the module namespace depending on the
+# Define all here to keep linters happy.  It should be overwritten by the code
+# below, but if in the future __all__ is not defined in httplib2 this will keep
+# things from breaking.
+__all__ = []
+
+# Import the proper implementation into the module namespace depending on the
 # current python version.  httplib2 supports python 2/3 by forking the code rather
 # than with a single cross-compatible module. Rather than modify third party code,
 # we'll just import the appropriate branch here.
 if six.PY3:
-    from .python3 import *
-    from .python3 import socks  # ensure include in namespace
-    import ssl
-    ssl_error_classes = (ssl.SSLError, ssl.CertificateError)
+    # Generate ssl_error_classes
+    import ssl as __ssl
+    ssl_error_classes = (__ssl.SSLError, __ssl.CertificateError)
+    del __ssl
+
+    # get the python3 fork of httplib2
+    from . import python3 as __httplib2_compat
+
+
 else:
-    from .python2 import *
-    from .python2 import socks  # ensure include in namespace
-    from .python2 import SSLHandshakeError  # TODO: shouldn't rely on this. not public
-    ssl_error_classes = (SSLHandshakeError,)
+    # Generate ssl_error_classes
```

Comment:
Could you add a test demonstrating that nothing is missing?

Reply:
Added a test.  Let me know if you'd like to see more covered, or if you think I should move it to a different location (I added it in test_api, since I didn't want to add a whole new file for just one test.)